Svenbjorn/visitfaroeislands.com
# weather-map-api-key

Monday: https://visitfaroeislands.monday.com/boards/4732496989/pulses/5407309326
Open weather map api key should be changed to one that visit faroe islands owns, instead of VITA owning the key.
In the process is a api keys storage vault made, where an appropriate API Keys get class has been made.

## How to test

Update the vault by changing or creating the following file: "/vault/apiKeys.json"
No changes should be in the solution, but you may confirm that the weather info comes into the site as expected. This can be with the top header menu in a specific module that shows the weather info

 - Code review the solution
